src/EnergyPlus/EconomicLifeCycleCost.hh

```diff
@@ -356,41 +356,41 @@ namespace EconomicLifeCycleCost {
     struct UsePriceEscalationType
     {
         // Members
-        std::string name;                // Name
-        Constant::ResourceType resource; // resource like electricity or natural gas (uses definitions from DataGlobalConstants)
-        int escalationStartYear;         // Escalation Start Year 1900-2100
-        int escalationStartMonth;        // Escalation Start Month 1 to 12
-        Array1D<Real64> Escalation;      // Escalation by year, first year is baseDateYear
+        std::string name;             // Name
+        Constant::eResource resource; // resource like electricity or natural gas (uses definitions from DataGlobalConstants)
+        int escalationStartYear;      // Escalation Start Year 1900-2100
+        int escalationStartMonth;     // Escalation Start Month 1 to 12
+        Array1D<Real64> Escalation;   // Escalation by year, first year is baseDateYear
         // last year is baseDateYear + lengthStudyYears - 1
 
         // Default Constructor
-        UsePriceEscalationType() : resource(Constant::ResourceType::None), escalationStartYear(0), escalationStartMonth(0)
+        UsePriceEscalationType() : resource(Constant::eResource::Invalid), escalationStartYear(0), escalationStartMonth(0)
```

Comment:
@dareumnam on the next PR round, could you just remove these default constructors, and instead simply put the assignments up where the variables are declared a few lines up?